### ISSUE TITLE
Workshops

### ISSUE THREAD

**Issue by Canvas02** [microsoft/vscode#129827]
When working in VScode I'll have a lot of extensions and a lot of them i don't use every time, So I'll be working with Rust and have the Deno extension running in the background, So a feature were we can create templates (workshops) where we can say use these extensions, disable these extensions, and keep these extensions at all times for a certain workshop will be really good for speed because recently I needed to use another text editor because VScode got too slow

In short, a feature were we can make templates (workshops) and say use these extensions, disable these, and keep these at all times will be good for performance

**Comment by gjsjohnmurray** [comment]
#116740 is probably related.

Pinging @misolori in case the term "workshops" is of interest.

**Comment by Canvas02** [comment]
@gjsjohnmurray do I close this issue?

**Comment by gjsjohnmurray** [comment]
@Canvas02 yes I think it can be considered a duplicate. Thanks!
